Description: Do not embed sqlite3 when building. Use debian
 version instead.
Author: Ruben Undheim <ruben.undheim@gmail.com>
Index: osmotrx/Makefile.am
===================================================================
--- osmotrx.orig/Makefile.am	2015-02-06 23:31:31.316944097 +0100
+++ osmotrx/Makefile.am	2015-02-07 13:01:36.197792681 +0100
@@ -28,7 +28,6 @@
 
 # Order must be preserved
 SUBDIRS = \
-	sqlite3 \
 	CommonLibs \
 	GSM \
 	Transceiver52M
Index: osmotrx/Makefile.common
===================================================================
--- osmotrx.orig/Makefile.common	2015-02-06 23:31:31.316944097 +0100
+++ osmotrx/Makefile.common	2015-02-07 13:01:19.373792297 +0100
@@ -33,6 +33,6 @@
 
 COMMON_LA = $(top_builddir)/CommonLibs/libcommon.la
 GSM_LA = $(top_builddir)/GSM/libGSM.la
-SQLITE_LA = $(top_builddir)/sqlite3/libsqlite.la -ldl
+SQLITE_LA = -lsqlite3
 
 MOSTLYCLEANFILES = *~
